util/exec.go

```diff
@@ -56,11 +56,11 @@ func CommandWithResult(ctx context.Context, logger *log.Logger, command *Command
 	return writer.String(), nil
 }
 
-// FilterArgs parses args and removes the ignored ones.
-func FilterArgs(args []string, ignored []string) (filtered []string) {
+// FilterArgs filters the arguments by either ignoring/allowing them in the result.
+func FilterArgs(args []string, filter []string, ignore bool) (filtered []string) {
```

Comment:
boolean arguments are not always ideal... cause they are not really visible from the caller maybe add two helpers `FilterArgsRemove` and `FilterArgsKeep` that then call this helper with the boolean set